jenkins/ocp-config/deploy/jenkins-master.yml

```diff
@@ -193,6 +196,8 @@ objects:
                 secretKeyRef:
                   key: auth-token
                   name: sonarqube
+            - name: SONARQUBE_EDITION
```

Comment:
@renedupont @michaelsauter Note in `ods-core.env` the name of the variable is `SONAR_EDITION` but *within* Jenkins it is `SONARQUBE_EDITION`.